api_docs/running-bots.md

```diff
@@ -20,6 +20,8 @@ You'll need:
 
 **Note: Please be considerate when testing experimental bots on public servers such as chat.zulip.org.**
 
+{start_tabs}
```

Comment:
This block of instructions could use a little clean-up while we're here. In particular, the last three steps all have bits that we could improve.

Reply:
Yes, I agree. Hmm, I need to think a bit more about how to clean this one up.

Reply:
![image](https://github.com/zulip/zulip/assets/2343554/fb3a5dfe-805a-43ca-bf68-f2563c237dee)